skimage/exposure/tests/test_exposure.py

```diff
@@ -276,7 +276,8 @@ def test_adapthist_grayscale():
     img = skimage.img_as_float(data.astronaut())
     img = rgb2gray(img)
     img = np.dstack((img, img, img))
-    with expected_warnings(['precision loss|non-contiguous input']):
+    with expected_warnings(['precision loss|non-contiguous input',
+                            r'divide by zero|\A\Z']):
```

Comment:
Err, that doesn't look right.  We should not be dividing by zero!

Reply:
haha good point!

Reply:
I removed this change allowing this PR to go forward. Somebody will have to investigate this warning.